pkg/image/controller/controller.go

```diff
@@ -163,7 +169,50 @@ func (c *ImportController) Next(stream *api.ImageStream, notifier Notifier) erro
 		if apierrs.IsNotFound(err) && client.IsStatusErrorKind(err, "imageStream") {
 			return ErrNotImportable
 		}
-		glog.V(4).Infof("Import stream %s/%s partial=%t error: %v", stream.Namespace, stream.Name, partial, err)
+		glog.V(4).Infof("Import stream %s/%s partial=%t error: %#+v", stream.Namespace, stream.Name, partial, err)
+		if apierrs.IsInvalid(err) && client.IsStatusErrorKind(err, "ImageStreamImport") {
```

Comment:
Yes, please open a new issue and I will start working on it right after this merges.